sdk/documenttranslation/azure-ai-documenttranslation/tests/test_translation.py

```diff
@@ -156,4 +156,168 @@ def test_single_source_single_target_with_suffix(self, client):
         ]
 
         # submit job and test
-        self._submit_and_validate_translation_job(client, translation_inputs, 1)
\ No newline at end of file
+        self._submit_and_validate_translation_job(client, translation_inputs, 1)
+
+    @DocumentTranslationPreparer()
+    @DocumentTranslationClientPreparer()
+    def test_bad_input_source(self, client):
+        # prepare containers and test data
+        target_container_sas_url = self.create_target_container()
+
+        # prepare translation inputs
+        translation_inputs = [
+            DocumentTranslationInput(
+                source_url="https://idont.ex.ist",
+                targets=[
+                    TranslationTarget(
+                        target_url=target_container_sas_url,
+                        language_code="es"
+                    )
+                ]
+            )
+        ]
+
+        with pytest.raises(HttpResponseError) as e:
```

Comment:
does Python check that error code is returned and things like that?

Reply:
This currently doesn't raise. Once the work for #17344 is in then I'll update it.